client/src/view/EditProfile.js

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import styled from 'styled-components';
+import User from "./icons/User";
+
+const Wrapper = styled.div`
+	font-weight: bold;
+	width: 70%;
+	height: 100%;
+`
+const Heading = styled.div`
+	color:  #004d99;
+	font-size: 1.3em;
+	border-bottom: 1px solid #004d99;
+	margin: 15px;
+`
+
+const Flex = styled.div`
+  display: flex;
+  border-bottom: 1px solid red;
+`
+
+const Input = styled.input`
+	display: block;
+	background-color: #E8E8E8;
+	width: 235px;
+	height: 25px;
+	margin: 6px;
+	font-size: 14px;
+	color: #A9A9A9;
+	border: none;
+`
+
+const Label = styled.label`
+	width: 200px;
+	display: inline-block;
+	text-align: left;
+	font-family: Arial, Helvetica, sans-serif;
+	
+`
+
+const Field = styled.div`
+	width: 200px;
+	display: inline-block;
+	text-align: left;	
+
+`
+
+const Checkbox = styled.input`
```

Comment:
This might look wierd but if you replaced this line with this
```jsx
const Checkbox = styled(props => <input type="checkbox" {...props} />)`

`
```

This will stop you from having to do 

```jsx
<Checkbox type="checkbox" name="sms"/>
```

You will be able to do just
```jsx
<Checkbox name="sms"/>
```